packages/manfred2word/src/engine/doc-parts/language-section/language-section.constants.ts

```diff
@@ -0,0 +1,16 @@
+import { LanguageType, LevelLanguageType } from './language-section.vm';
+
+export const languageList: LanguageType[] = [
+  { iso: 'es', name: 'Español' },
+  { iso: 'it', name: 'Italiano' },
+  { iso: 'en', name: 'Inglés' },
+  { iso: 'gl', name: 'Gallego' },
```

Comment:
add more languages, such as French, German, Chinese, ...

here you have a website where you can find the codes
[https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes)

insert the most used languages